azure-pipelines.yml

```
@@ -489,3 +519,38 @@ stages:
+    jobs:
+      - job: PUBLISH
+        displayName: 'Publishing'
+        timeoutInMinutes: "600"
+        steps:
+          - checkout: self
+            fetchDepth: 1
+            clean: false
+            persistCredentials: true
+          - script: |
+              set -ex
+                            
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)